e2e/tests/hmctsSuperUserAdministersCase_test.js

```diff
@@ -5,6 +5,8 @@ const caseManagementCaseData = require('../fixtures/caseData/prepareForHearing.j
 const closedCaseData = require('../fixtures/caseData/closedCase.json');
 const orderFunctions = require('../helpers/generated_order_helper');
 const blankOrder = require('../fixtures/orders/blankOrder.js');
+const finalHearingCaseData = require('../fixtures/caseData/finalHearing.json');
+const moment = require('moment');
```

Comment:
We have the dateFormat library so why do we have moment as well? Is it a transitive dependancy from another package?

Reply:
Just looked and it was added recently. Wonder why